Dockerfile

```diff
@@ -4,7 +4,11 @@ WORKDIR     /dashdotdb
 
 COPY        . ./
 
+USER        root
```

Comment:
not really a fan of this when we need to run the container without root permissions in Openshift. Apparently there's a bug in the last image, this is how I solved this for github-mirror

https://github.com/app-sre/github-mirror/blob/master/Dockerfile#L1

Reply:
Good point, fully agree!
Personally, for me in an ideal world we pin all base images and rely on sth like renovatebot for weekly base image upgrade PRs. But thats gonna be a longer story :)

Adjusted it now as suggested.